src/Action_Closest.cpp

```diff
@@ -318,11 +317,25 @@ Action::RetType Action_Closest::DoAction(int frameNum, ActionFrame& frm) {
       V_atom_coords_[index + 2] = a[2];
     }
   }
-  // TODO handle imaging
+  
+
+  //low overhead so we will do it anyway
+  Box frmBox = frm.Frm().BoxCrd();
+  Matrix_3x3 ucell, recip;
+  frm.Frm().BoxCrd().ToRecip(ucell, recip);
+  double box[3] = {frmBox[0], frmBox[1],frmBox[2]};
```

Comment:
@amit56r Any particular reason you are copying box coordinates and unit cell/fractional matrices to temporary arrays instead of using the underlying data pointers (i.e. Box::boxPtr() and Matrix_3x3::Dptr())?